lib/std/private/globs.nim

```diff
@@ -45,10 +45,15 @@ iterator walkDirRecFilter*(dir: string, follow: proc(entry: PathEntry): bool = n
 
 proc nativeToUnixPath*(path: string): string =
```

Comment:
__[As I](https://github.com/nim-lang/Nim/issues/18142#issuecomment-851889322)__ (and __[others](https://github.com/nim-lang/Nim/pull/13265#issuecomment-674700064)__) have mentioned elsewhere, this is not a function that should be used. It *cannot* accurately translate paths without either a loss of information, or ambiguity. Windows has a __[couple of different path types](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats)__ and using a function that attempts to convert them all to some standard format is not going to work. Please stop pushing this.

Reply:
>  It cannot accurately translate paths without either a loss of information, or ambiguity.

cygwin works just fine, which is almost what I'm doing. future work will follow more closely what `cygpath` does, so that a spec is followed, you can't argue with that (I'm currently almost doing the same).